cmd/kpod/load.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"io"
+	"os"
+
+	"io/ioutil"
+
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	dockerArchive = "docker-archive:"
+	oci           = "oci:"
+)
+
+type loadOptions struct {
+	input string
+	quiet bool
+}
+
+var (
+	loadFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "input, i",
+			Usage: "Read from archive file, default is STDIN",
+			Value: "/dev/stdin",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "Suppress the output",
+		},
+	}
+	loadDescription = "Loads the image from docker-archive by default or from oci if the oci flag is set."
```

Comment:
There is no OCI flag.  Remove this qualifier.  I think for now we should remove oci from this, until we have an oci-archive.  It seems that oci transport creates a directory structure not a tar ball.